src/Gaffer/Context.cpp

```diff
@@ -477,3 +477,40 @@ const std::string &Context::SubstitutionProvider::variable( const boost::string_
 	m_formattedString.clear();
 	return m_formattedString;
 }
+
+// Forward compatibility with Gaffer 0.60
+void Context::EditableScope::setAllocated( const IECore::InternedString &name, const IECore::Data *value )
+{
+	set( name, value );
+}
+
+void Context::EditableScope::setFramesPerSecond( const float *framesPerSecond )
+{
+	setFramesPerSecond( *framesPerSecond );
+}
+
+IECore::DataPtr Context::getAsData( const IECore::InternedString &name ) const
+{
+	const IECore::Data* result = get<Data>( name );
+	if( result )
+	{
+		return result->copy();
+	}
+	else
+	{
+		return nullptr;
+	}
```

Comment:
Shouldn't this throw if it can't get the data? Oh, I see from the tests that it must be throwing somehow, so I guess that happens in `get()` and the `if( result )` conditional is unnecessary?